actions/bump-and-release-cloud-plugin/action.yaml

```diff
@@ -0,0 +1,320 @@
+        cp ${{ steps.metadata.outputs.archive }} ${{ steps.metadata.outputs.archive-latest }}
+        md5sum ${{ steps.metadata.outputs.archive-latest }} > ${{ steps.metadata.outputs.archive-latest-checksum }}
+        cd ..
+
+    - name: Validate plugin with validator config
+      if: inputs.validator-config-path != ''
+      shell: bash
+      run: |
+        npx -y @grafana/plugin-validator@latest -config ${{ inputs.validator-config-path }} ${{ runner.temp }}/plugin-artifacts/${{ steps.metadata.outputs.archive }}
+
+    - name: Validate plugin
+      if: inputs.validator-config-path == ''
+      shell: bash
+      run: |
+        npx -y @grafana/plugin-validator@latest ${{ runner.temp }}/plugin-artifacts/${{ steps.metadata.outputs.archive }}
+
+    # Now that we've validated our plugin we can safely commit the version bump
+    - name: Commit version bump
+      id: commit-version-bump
+      uses: grafana/github-api-commit-action@ccf9b520c5698380ad3b9619c5add427369b7ef1 # v0.2.0
+      with:
+        token: ${{ inputs.github-token }}
+        commit-message: 'chore(version): bump to version [${{ steps.output-version.outputs.version }}] (${{ inputs.release-level }}) [BOT]'
+
+    - name: Create tag
+      uses: rickstaa/action-create-tag@a1c7777fcb2fee4f19b0f283ba888afa11678b72 # v1.7.2
+      with:
+        commit_sha: ${{ steps.commit-version-bump.outputs.commit-sha }}
+        github_token: ${{ inputs.github-token }}
+        message: "chore(tag): tag version ${{ steps.output-version.outputs.version-tag }} [BOT]"
+        tag: ${{ steps.output-version.outputs.version-tag }}
+
+    - name: Upload build artifact
+      uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3 # v4.3.1
+      with:
+        name: upload-dir
+        path: ${{ runner.temp }}/plugin-artifacts
+
+    # Checkout the tag, to create the release
+    - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        ref: ${{ steps.output-version.outputs.version-tag }}
+
+    - name: Create Github release
+      uses: softprops/action-gh-release@9d7c94cfd0a1f3ed45544c887983e9fa900f0564 # v2.0.4
+      with:
+        prerelease: ${{ inputs.release-level == 'prerelease' }}
+        generate_release_notes: true
+        files: |
+          ${{ runner.temp }}/plugin-artifacts/**/*.zip?(.md5)
+        tag_name: ${{ steps.output-version.outputs.version-tag }}
+        token: ${{ inputs.github-token }}
+
+    - id: auth
+      name: Authenticate to Google Cloud
+      uses: google-github-actions/auth@55bd3a7c6e2ae7cf1877fd1ccb9d54c0503c457c # v2.1.2
+      with:
+        credentials_json: '${{ inputs.gcs-service-account-creds }}'
+
+    - id: upload-to-gcs
+      name: Upload assets
+      uses: google-github-actions/upload-cloud-storage@22121cd842b0d185e042e28d969925b538c33d77 # v2.1.0
+      with:
+        path: ${{ runner.temp }}/plugin-artifacts
+        destination: ${{ inputs.gcs-bucket }}
+        parent: false
```

Comment:
Is it possible to add new steps to optionally create deployment_tools PRs to bump version in dev | ops | prod ... env automatically? [this is an example](https://github.com/grafana/grafana-aws-app/blob/5c04bc1fd3b3604b9e0e03e9fffe52cb62a0925a/.github/workflows/release.yml#L189)